get.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sudo apt-get update && sudo apt-get install -y curl
```

Comment:
Suggest to check if curl command exist to prevent unnecessary procedures.

```
if ! [ -x "$(command -v curl)" ]; then
  sudo apt-get update && sudo apt-get install -y curl
fi
```

Reply:
Fixed, thanks.